glusterd2/brick/types.go

```diff
@@ -32,6 +32,13 @@ type MountInfo struct {
 	MntOpts        string
 }
 
+type DeviceInfo struct {
```

Comment:
Linter is failing here. Add a description  for this struct.

Reply:
fixed. Also fixed issues related to deviceinfo detail update during snapshot create and clone